packages/help-text/README.md

```diff
@@ -1,4 +1,4 @@
-## Description
+## Overview
```

Comment:
If we're going to change this from Description to Overview, should we amend the template for our READMEs? Not something to do in this PR, necessarily, just calling it out.

Reply:
@najikahalsema I am in the process of updating them [here](projects/documentation/content/guides/adding-component.md). Is there somewhere else I should also update?

Reply:
Oh, no, if you're working on it somewhere else, that's fine! Just wasn't aware of that :^)